packages/qwik-city/buildtime/vite/image-jsx.ts

```diff
@@ -111,8 +112,31 @@ export function optimizeSvg(
   userOpts?: QwikCityVitePluginOptions
 ) {
   const svgAttributes: Record<string, string> = {};
-  // note: would be great if it warned users if they tried to use qwik-default plugins, so that name collisions are avoided
-  const userPlugins = userOpts?.imageOptimization?.svgo?.plugins || [];
+  const prefixIdsConfiguration = userOpts?.imageOptimization?.svgo?.prefixIds;
+  const maybePrefixIdsPlugin: SVGOConfig['plugins'] =
+    prefixIdsConfiguration !== false ? [{ name: 'prefixIds', params: prefixIdsConfiguration }] : [];
+
+  const userPlugins =
+    userOpts?.imageOptimization?.svgo?.plugins?.filter((plugin) => {
+      if (
+        plugin === 'preset-default' ||
+        (typeof plugin === 'object' && plugin.name === 'preset-default')
+      ) {
+        console.warn(
+          `You are trying to use the preset-default SVGO plugin. This plugin is already included by default, you can customize it through the defaultPresetOverrides option.`
+        );
+        return false;
+      }
+
+      if (plugin === 'prefixIds' || (typeof plugin === 'object' && plugin.name === 'prefixIds')) {
+        console.warn(
+          `You are trying to use the preset-default SVGO plugin. This plugin is already included by default, you can customize it through the prefixIds option.`
```

Comment:
Contains a typo though :/ The checked plugin name is "prefixIds", not "preset-default"